<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.0 (Python 3.12.2 on win32)" generated="2024-04-28T01:49:05.864137" rpa="false" schemaversion="5">
<suite id="s1" name="Automation" source="C:\Users\Shubham Pandey\PycharmProjects\Automation">
<suite id="s1-s1" name="Practice" source="C:\Users\Shubham Pandey\PycharmProjects\Automation\Practice">
<suite id="s1-s1-s1" name="Demo1" source="C:\Users\Shubham Pandey\PycharmProjects\Automation\Practice\Demo1.robot">
<test id="s1-s1-s1-t1" name="Validate opening of browser" line="7">
<kw name="Open youtube">
<msg time="2024-04-28T01:49:06.069545" level="FAIL">No keyword with name 'Open youtube' found.</msg>
<arg>chrome</arg>
<status status="FAIL" start="2024-04-28T01:49:06.069545" elapsed="0.000000">No keyword with name 'Open youtube' found.</status>
</kw>
<kw name="play Ankhiyaan gulab">
<status status="NOT RUN" start="2024-04-28T01:49:06.071552" elapsed="0.000000"/>
</kw>
<status status="FAIL" start="2024-04-28T01:49:06.067543" elapsed="0.005000">No keyword with name 'Open youtube' found.</status>
</test>
<doc>First program</doc>
<status status="FAIL" start="2024-04-28T01:49:05.884168" elapsed="0.192375"/>
</suite>
<status status="FAIL" start="2024-04-28T01:49:05.883199" elapsed="0.198366"/>
</suite>
<status status="FAIL" start="2024-04-28T01:49:05.866183" elapsed="0.218373"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="Automation">Automation</stat>
<stat pass="0" fail="1" skip="0" id="s1-s1" name="Practice">Automation.Practice</stat>
<stat pass="0" fail="1" skip="0" id="s1-s1-s1" name="Demo1">Automation.Practice.Demo1</stat>
</suite>
</statistics>
<errors>
<msg time="2024-04-28T01:49:06.066543" level="ERROR">Error in file 'C:\Users\Shubham Pandey\PycharmProjects\Automation\Practice\Demo1.robot' on line 4: Resource file 'C:UsersShubham PandeyPycharmProjectsAutomationResourcesbusiness_keywords.robot' does not exist.</msg>
</errors>
</robot>
